### PR TITLE
Fix mixed content warnings

### PR DESCRIPTION
**Pull request by mathiasbynens**
This makes https://cheeaun.github.io/tweets/ work over HTTPS without triggering any mixed content warnings.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
+				<a href="#" id="results-more" class="more-link hide">More&hellip;</a>
+			</div>
+		</div>
+	</div>
+	<div id="lightbox-cover"></div>
+	<div id="lightbox"></div>
+	<script src="config.js"></script>
+	<script src="assets/js/scripts.min.js"></script>
+	<script src="assets/js/app.js"></script>
+</body>
```
Comment (mathiasbynens) [inline index.html]: Looks like these files weren’t using LF line endings.

Comment (cheeaun) [inline index.html]: :open_mouth: Oh, didn't know that. Thought Atom editor would auto-fix it for me :confused:

**Comment by mathiasbynens** [comment]
Thanks for merging! Now you can update this URL to the HTTPS version, if you like:

image

**Comment by cheeaun** [comment]
:+1: updated, thanks!

**Pull request merged**